### PR TITLE
DCOS-6185: Assign flexible dimensions to alert panels

### PR DESCRIPTION
**Pull request by jfurrow**

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @mlunoe

**Comment by mlunoe** [comment]
BOOM! 💥

File: src/js/components/AlertPanel.js [hunk 1]
```diff
@@ -1,9 +1,7 @@
 var classNames = require('classnames');
 var React = require('react');
-var ReactDOM = require('react-dom');
 
 var InternalStorageMixin = require('../mixins/InternalStorageMixin');
```
Comment (kennyt) [inline src/js/components/AlertPanel.js]: you can remove this now that you don't use it

**Comment by kennyt** [comment]
after my one comment, :ship:

File: src/js/components/AlertPanel.js (hunk 1 above)
Comment (jfurrow) [inline src/js/components/AlertPanel.js]: Good call, gone!

**Pull request merged**